fabric-chaincode-integration-test/getDockerImages.sh

```diff
@@ -1,43 +1,22 @@
-#!/bin/bash -e
+#!/bin/bash
+#
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
 set -euo pipefail
 
-echo "======== PULL DOCKER IMAGES ========"
+version=${FABRIC_VERSION:-2.1}
+artifactory_url=hyperledger-fabric.jfrog.io
 
-###############################################################
-# Pull and Tag the fabric and fabric-ca images from Artifactory
-###############################################################
-echo "Fetching images from Artifactory"
-ARTIFACTORY_URL=hyperledger-fabric.jfrog.io
-ORG_NAME="hyperledger"
+for image in peer orderer ca tools orderer ccenv; do
```

Comment:
Do you actually require `ccenv` for anything? Or can it be removed

Reply:
I think it's a hangover from the ancestor of this script so it can be removed.